apis/routes/wishRouter.js

```diff
@@ -0,0 +1,9 @@
+const router = require('express').Router();
+const { wishController } = require('../controllers');
+const { loginRequired } = require("../utils/auth");
+
+router.post('', loginRequired, wishController.createWish);
+router.get('', loginRequired, wishController.getWishList)
+router.delete('/:productId', loginRequired, wishController.removeWish);
```

Comment:
query parameter로 productId 받기
```suggestion
router.delete('', loginRequired, wishController.removeWish);
```